TPMCmd/tpm/include/TpmASN1.h

```diff
@@ -85,13 +85,6 @@
 //** Macros
 
 //*** Unmarshaling Macros
-#ifndef VERIFY
```

Comment:
Why removed?

Reply:
Is this okay, or should I fix this definition of `VERIFY` instead of removing it?

Reply:
If it compiles having only one copy in GpMacros, then this is a good find we should leave it out.  When I saw the change I didn't realize it was in the ASN1 header and thought you were deleting it out of GpMacros.

Reply:
It does indeed.